pkg/gql/query/blocks.go

```diff
@@ -207,10 +187,76 @@ func resolveTxs(p graphql.ResolveParams) (interface{}, error) {
 
 		return txs, err
 	}
-
 	return nil, errors.New("invalid source block")
 }
 
+func resolveReward(p graphql.ResolveParams) (interface{}, error) {
+	b, ok := p.Source.(*queryHeader)
+	if ok {
+		// Retrieve DB conn from context
+		db, ok := p.Context.Value("database").(database.DB)
+		if !ok {
+			return nil, errors.New("context does not store database conn")
+		}
+
+		reward := uint64(0)
+
+		err := db.View(func(t database.Transaction) error {
+			fetched, err := t.FetchBlockTxs(b.Hash)
```

Comment:
Any reason to fetch txs from database as you have them already in `b.Txs`?

```
for _, tx := range b.Txs {
...
```

Reply:
During the resolve of this graphql node, the transactions are not resolved yet, so they are not valued (this was the issue that this PR is going to solve)